Specs/Request/Request.JSONP.js

```diff
@@ -52,4 +57,29 @@ describe('Request.JSONP', function(){
 
 	});
 
+	it('should clear the request callback map', function(){
```

Comment:
@DimitarChristoff: we are still using Jasmine `1.3` and the `done()` API is from `2.0`. We can do that upgrade in another PR, that would be good.